lib/pokemon-site.js

```diff
@@ -24,11 +24,21 @@ class PokemonSite {
     this._app = app
 
     // Start listening
-    const server = http.createServer(app)
-    server.listen(config.listenPort, config.listenAddress, () => {
+    this._server = http.createServer(app)
+    this._server.listen(config.listenPort, config.listenAddress, () => {
       console.log('Listening on %s:%d ...', config.listenAddress, config.listenPort)
     })
-    this._server = server
+
+    // Keep track of open connections
+    this.socketId = 0
+    this.sockets = {}
+    this._server.on('connection', (socket) => {
+      let socketId = ++this.socketId
```

Comment:
No. But I suppose there is also no reason why it would start with 0, or is there?
But anyways, if this is important to you I accept your wish for indices starting with 0. See the latest commit.